spec/unit/type/schedule_spec.rb

```diff
@@ -501,6 +506,11 @@ def min(method, count)
       expect(@schedule.match?).to be_truthy
     end
 
+    it "should match if weekday is 1" do
+      @schedule[:weekday] = 1
+      expect(@schedule.match?).to be_truthy
```

Comment:
nit, for rspec will produce a more meaningful message on failure if you write this as:

```ruby
expect(@schedule).to be_match
```

Otherwise if the expectation fails, it will just say false doesn't equal true.